client/sites/tools/sftp-ssh/page.tsx

```diff
@@ -0,0 +1,48 @@
+import { useTranslate } from 'i18n-calypso';
+import NavigationHeader from 'calypso/components/navigation-header';
+import Notice from 'calypso/components/notice';
+import { SftpForm } from 'calypso/sites/tools/sftp-ssh/sftp-form';
+import useIsSftpSshSettingSupported from './hooks/use-is-sftp-ssh-setting-supported';
+import useSftpSshSettingTitle from './hooks/use-sftp-ssh-setting-title';
+import { SftpCardLoadingPlaceholder } from './sftp-card-loading-placeholder';
+
+import './style.scss';
+
+function Container( { isLoading, children }: { isLoading: boolean; children: React.ReactNode } ) {
+	return isLoading ? <SftpCardLoadingPlaceholder /> : children;
+}
+
+function Description( { children }: { children?: React.ReactNode } ) {
+	const title = useSftpSshSettingTitle();
+	return <NavigationHeader title={ title } subtitle={ children } />;
+}
+
+export default function SftpSsh() {
+	const translate = useTranslate();
+
+	const isSupported = useIsSftpSshSettingSupported();
+	if ( isSupported === null ) {
+		return null;
+	}
+
+	const renderNotSupportedNotice = () => {
+		return (
+			<>
+				<Description />
+				<Notice showDismiss={ false } status="is-warning">
+					{ translate( 'This setting is not supported for this site.' ) }
+				</Notice>
+			</>
+		);
+	};
+
+	const renderSetting = () => {
+		return <SftpForm ContainerComponent={ Container } DescriptionComponent={ Description } />;
+	};
+
+	return (
+		<div className="tools-sftp-ssh">
+			{ isSupported ? renderSetting() : renderNotSupportedNotice() }
```

Comment:
super ![badge](https://img.shields.io/badge/review-nits-green.svg)
Would it be worth renaming this in a declarative way? 
```suggestion
			{ isSupported ? <Setting/> : <NotSupportedNotice/> }
```

Reply:
Thanks for the review! It's still in flux and will likely change if we want to show upsell instead of notice, so I'll hold off for now 👍